cypress/support/fragments/check-out-actions/check-out-actions.js

```diff
@@ -90,5 +90,33 @@ export default {
 
   checkItem:(barcode) => {
     cy.expect(MultiColumnList({ id:'list-items-checked-out' }).find(HTML(including(barcode))).absent());
-  }
+  },
+
+  confirmMultipieceCheckOut(barcode) {
+    cy.do(Button({ className: 'button---kpbBz interactionStyles---eaRy_ primary---xHTjI' }).click());
```

Comment:
There is bad way for selection element by this class name because letters after --- can be diffrent on other environment
For example, this test probably will be failed on snapshot, snapshot-2 environment
Try to use another selector or interactor please